Create german.json

markusmarkusz opened the pull request:
Added some parts of the german translation

markusmarkusz committed:
Create german.json

Nayruden commented on locale/core/german.json:
```diff
@@ -0,0 +1,37 @@
+	"MINUTE_SHORT": "m",
+	"HOURS": "Stunden",
+	"HOUR": "Stunde",
+	"HOUR_SHORT": "h",
```
Should this be 'S'? This is a one-character way to refer to hours, so in English, "12h" would mean 12 hours. I realize that 's' is taken by 'seconds', so does 'S' make more sense here? In context, in German, "12S 15m 19s" would mean 12 hours, 15 minutes, 19 seconds.

Nayruden commented on locale/core/german.json:
```diff
@@ -0,0 +1,37 @@
+	"HOUR_SHORT": "h",
+	"DAYS": "Tage",
+	"DAY": "Tag",
+	"DAY_SHORT": "d",
```
Should be 'T'?

Nayruden commented on locale/core/german.json:
```diff
@@ -0,0 +1,37 @@
+	"MONTH_SHORT": "M",
+	"YEARS": "Jahre",
+	"YEAR": "Jahr",
+	"YEAR_SHORT": "y",
```
Should be 'J'?

Nayruden commented on locale/core/german.json:
```diff
@@ -0,0 +1,37 @@
+
+	"ARG_NOT_SPECIFIED": "Es wurden nicht genügend Argumente angegeben.",
+	"ARG_CANNOT_PARSE": "Cannot parse '{GIVEN}' to {TYPE}",
+	"ARG_INCORRECT_TYPE": "Not a {TYPE} (given '{GIVEN}')",
```
No translations for ARG_CANNOT_PARSE and ARG_INCORRECT_TYPE?

markusmarkusz replied:
Give me an example sentence. Then I can better translate it ^^.

Nayruden replied:
Cannot parse: "Cannot parse 'bob' to number" or "Cannot parse 'table: 0x7fe400512690' to string"

Incorrect type: "Not a number (given 'bob')" or "Not a string (given 'table: 0x7fe400512690')"

Does that make sense?

Nayruden commented:
Very helpful, thank you! Though, we are at a *very* early stage of ULX4 development, so we will need your help in the future as more phrases get added. Can we contact you in the future regarding German translations?

I made a few comments on your translations; please take a look at them before I merge it.

markusmarkusz commented:
I hadn't add the short forms (w, m, M, s, ...) because I thought that
they weren't so important. I'll add them later.
Can you give me an example message for ARG_CANNOT_PARSE and
ARG_INCORRECT_TYPE? Then I can translate it better.

And yes you can contact me for future translations to German.

markusmarkusz committed:
German Translation

Added the short forms.
ARG_CANNOT_PARSE and ARG_INCORRECT_TYPE are translated if I have a example sentence.

markusmarkusz committed:
German Translation
